routers/api/v1/repo/issue.go

```diff
@@ -67,20 +67,24 @@ func SearchIssues(ctx *context.APIContext) {
 
 	// find repos user can access (for issue search)
 	repoIDs := make([]int64, 0)
+	opts := &models.SearchRepoOptions{
+		PageSize:    15,
+		Private:     false,
```

Comment:
I think this might still need to be 
```suggestion
		Private:     true,
```

Reply:
@zeripath your suggestion would allow anon to look at private repos!

for loged in users this is set to **true** at:
 -> https://github.com/go-gitea/gitea/pull/9698/files#diff-6a82b880ff1ae1c2e9b577046e45a6cbR79-R83